### PR TITLE
Local test run takes 15 second

### PR DESCRIPTION
**Pull request by sergioboi**
In the end of test run we try to send data to report. It makes test run longer on 15 second

### CODE REVIEW

**Comment by sergioboi** [comment]
Before
Снимок экрана 2020-02-08 в 16 58 57

**Comment by sergioboi** [comment]
After
Снимок экрана 2020-02-08 в 16 59 36

File: subprojects/android-test/test-app/build.gradle.kts [hunk 1]
```diff
@@ -39,11 +39,10 @@ android {
...
                 "sentryDsn" to "http://stub-project@stub-host/0",
-                "fileStorageUrl" to "https://stub",
-                "reportApiUrl" to "https://stub",
-                "reportApiFallbackUrl" to "https://stub",
-                "reportViewerUrl" to "https://stub",
-                "unnecessaryUrl" to "https://localhost"
+                "fileStorageUrl" to "https://file-storage-stub",
+                "unnecessaryUrl" to "https://localhost",
+                // todo move to instrumentation plugin
+                "avito.report.enabled" to "false"
```
Comment (sergioboi) [inline subprojects/android-test/test-app/build.gradle.kts:45]: Feature is on Runner Level now cause of compatibility issues.

File: subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/TestRunEnvironment.kt [hunk 2]
```diff
@@ -96,14 +99,27 @@ fun provideEnvironment(
     return when (isFakeOrchestratorRun) {
         true -> TestRunEnvironment.OrchestratorFakeRunEnvironment
         false -> try {
+            val coordinates = ReportCoordinates(
+                planSlug = argumentsProvider.getMandatoryArgument("planSlug"),
+                jobSlug = argumentsProvider.getMandatoryArgument("jobSlug"),
+                runId = argumentsProvider.getMandatoryArgument("runId")
+            )
+            // todo move to instrumentation plugin
```
Comment (dsvoronin) [inline subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/TestRunEnvironment.kt:107]: local report will always be enabled?

**Review by dsvoronin — APPROVED** [review]

**Comment by dsvoronin** [comment]
` test-runs-slow `

https://www.youtube.com/watch?v=r2env-txnms

File: subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/TestRunEnvironment.kt (hunk 2 above)
Comment (sergioboi) [reply subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/TestRunEnvironment.kt:107]: If you have url u can enable it. If you don't - can disable

**Pull request merged**